### ISSUE TITLE
Bug: Manually updated ticker has deltaTime and deltaMS different from an autoStarted ticker.

### ISSUE THREAD

**Issue by smjnab** [pixijs/pixijs#9877]
### Current Behavior

Unsure if this is a bug, intended or something you are expected to do before updating a ticker the first time.

Prevent a ticker from starting, then manually call update instead. Doing this the first update will give you, if leaving targetFPS to default 60,  a delta of 6 and a ms value of 100. 

### Expected Behavior

That the values are the same as when a ticker is running on its own. Delta 1 and ms 16 or thereabouts at 60.

### Steps to Reproduce

Test at: https://codesandbox.io/p/sandbox/timermanuallystepped-6qn7fz
At the top I prevent the ticker from starting and at the bottom I have the requestAnimationFrame loop doing the manual update. I use the Application in the example, creating a new Ticker() does the same.

In short:

```
app.ticker.autoStart = false;
app.ticker.stop();

const manualRAF = (now) => {
    app.ticker.update(now);

    requestAnimationFrame(manualRAF);
 };

requestAnimationFrame(manualRAF);
```

### Environment

- **`pixi.js` version**: *7.3.2*
- **Browser & Version**: *Tested in Chrome and Safari*
- **OS & Version**: *latest macOS, Ubuntu 22.04*
- **Running Example**: *https://codesandbox.io/p/sandbox/timermanuallystepped-6qn7fz*

**Comment by smjnab** [comment]
OK, found the lastTime prop in the ticker. Sorry!
